vnext/JSI/Shared/ChakraApi.h

```diff
@@ -0,0 +1,465 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#ifdef CHAKRACORE
+#include "ChakraCore.h"
+#else
+#ifndef USE_EDGEMODE_JSRT
+#define USE_EDGEMODE_JSRT
+#endif
+#include <jsrt.h>
+#endif
+
+#include <cassert>
+#include <cstddef>
+#include <cstdint>
+#include <memory>
+#include <string>
+#include <string_view>
+
+// We use macros to report errors.
+// Macros provide more flexibility to show assert and provide failure context.
+
+// Check condition and crash process if it fails.
+#define ChakraVerifyElseCrash(condition, message)          \
```

Comment:
>ChakraVerifyElseCrash [](start = 8, length = 21)

Why not call it "VerifyElseCrash"?

Reply:
I do not want to have a conflict here. As we are going toward reduced project structure - the effort driven by Julio - we can replace this one with the VerifyElseCrash. I cannot do it now because this code has no dependency on the Mso library.

---
In reply to: [470235462](https://github.com/microsoft/react-native-windows/pull/5710#discussion_r470235462) [](ancestors = 470235462)